File: modulo1/segmento3/apunte7/ejercicio3/ejercicio3.py
def mayor(num1,num2,num3):
	
	if num1 >= num2:
		mayor= num1
	
	else:
		mayor=num2
	
	if num3>=mayor:
		mayor=num3
	
	else:
		mayor=mayor
	
	return mayor

num1=int(input("Ingrese un numero para que al finalizar se mostrara el numero mayor: "))
num2=int(input("Ingrese el segundo numero: "))
num3=int(input("Ingrese el tercer numero: "))

respuesta=mayor(num1,num2,num3)

print("El numero mayor es: "+ str(respuesta))

File: modulo1/segmento3/apunte7 "Funciones"/ejercicio3/ejercicio3.java
import java.util.Scanner;

public class ejercicio3{

	public static String mayor(Integer num1, Integer num2, Integer num3){

		String resultado ="";

		if(num1 == num2 && num1 == num3){
			System.out.println("Los tres numero son iguales: ");
		
		}else if(num1>num2 && num1>num3){
			System.out.println("El numero mayor es: "+num1);
		
		}else if(num2>num1 && num2>num3){
			System.out.println("El numero mayor es: "+num2);
		
		}else{
			System.out.println("El numero mayor es: "+num3);
		}
		return resultado;
	}

	public static void main(String args[]){
		Scanner teclado = new Scanner(System.in);
		String resultado="";
		
		System.out.println("Ingrese el primer numero: ");
		Integer num1 = Integer.parseInt(teclado.nextLine());
		
		System.out.println("Ingrese el segundo numero: ");
		Integer num2 = Integer.parseInt(teclado.nextLine());

		System.out.println("Ingrese el tercer numero: ");
		Integer num3 = Integer.parseInt(teclado.nextLine());

		resultado=mayor(num1, num2, num3);
		System.out.println(resultado);



	}
}


			
